src/DatePicker/DatePicker.js

```diff
@@ -150,6 +156,7 @@ class DatePicker extends Component {
 
   state = {
     date: undefined,
+    isFocused: false,
```

Comment:
That state isn't used in the `render` method. I don't think we need that value in the state. I would rather have it an instance property.